common/source/docs/common-polygon_fence.rst

```diff
@@ -1,64 +1,133 @@
 
+.. image:: ../../../images/fence_examples.png
+   :target: ../../_images/fence_examples.png
+
+[site wiki="copter"]
 Like the :ref:`cylindrical fence <common-ac2_simple_geofence>`, Copter will stop increasing altitude at :ref:`FENCE_ALT_MAX<FENCE_ALT_MAX>` in Loiter, PosHold, and AltHold modes. In other modes, it will execute the :ref:`FENCE_ACTION<FENCE_ACTION>` if the altitude limit is exceeded and active. 
+[/site]
+[site wiki="plane"]
+Like the :ref:`cylindrical fence <common-ac2_simple_geofence>`, Plane will execute the :ref:`FENCE_ACTION<FENCE_ACTION>` if the boundary or altitude limits are breached.
+[/site]
 
-See :ref:`cylindrical fence <common-ac2_simple_geofence>` for fence parameters, which also apply to these polygonal fences.
+These fences are created and treated in a similar way to mission command lists and rally point lists and loaded into the autopilot. Each item in the list specifies a point in the polygonal boundary or a circular item, whether its inclusion or exclusion type, and radius if circular.
+[site wiki="plane"]
+For Plane a fence breach return point can also be specified in the list.
+[/site]
 
-These fences are created and treated in a similar way to mission command lists and rally point lists and loaded into the autopilot.
+Multiple fences can be specified of differing types and shapes in the list.
 
 
 ..  youtube:: U3Z8bO3KbyM
     :width: 100%
 
+
+
 Enabling Fences in Mission Planner
 ==================================
 
-..  youtube:: SEm4nVfbg00
-    :width: 100%
+To enable Fences, go to the Mission Planner full parameter list (CONFIG->Full Parameter Tree), search for items with ``FENCE_``:
 
--  Connect your Autopilot to the Mission Planner
--  Go to the **CONFIG\| GeoFence** screen
--  Click "Enable" and set Type to "All" or another option that includes "Polygon". Circle refers to the HOME centered, :ref:`cylindrical fence <common-ac2_simple_geofence>`. Altitude only, limits maximum altitude (everywhere).
+.. image:: ../../../images/fence_enable.png
+    :target: ../_images/fence_enable.png
 
+Plane Parameter List (Copter/Rover have fewer)
 
-   .. image:: ../../../images/polygon_fence_enable.png
-       :target: ../_images/polygon_fence_enable.png
-       :width: 500px
+-  Set :ref:`FENCE_ACTION<FENCE_ACTION>` = to whatever you wish for a breach action. These will vary depending on vehicle type.
+-  Set :ref:`FENCE_ALT_MAX<FENCE_ALT_MAX>` = to the altitude limit you want (in meters). This is unavailable in Rover. A value of 0 disables this limit.
```

Comment:
Again, `FENCE_ALT_MAX = 0` does not disable the limit, and results in a breach if `FENCE_TYPE` has `ALT_MAX` set.

Reply:
fixed